scripts/zap.json

```diff
@@ -0,0 +1,9 @@
+{
+    "packages": [
+        {
+            "path": "fuchsia/third_party/zap/${platform}",
+            "platforms": ["linux-amd64", "linux-arm64", "mac-amd64", "windows-amd64"],
+            "tags": ["version:2@v2023.01.19-nightly.1"]
```

Comment:
This is going to need to change often.  Please update the zap version update script so it updates this too.

Reply:
Done! Thanks for pointing this out.